packages/circuit-ui/components/Selector/Selector.tsx

```diff
@@ -221,8 +258,20 @@ export const Selector = forwardRef(
           className={className}
           style={style}
         >
-          {children}
+          <Fragment>
+            {label || children}
+            {description && (
+              <FieldDescription aria-hidden="true">
```

Comment:
I considered that approach as well, but that because complicated quickly because you need to calculate how much space the description needs to reserve enough space inside the box.